backend/notification-service/src/main/java/com/furniture/notificationservice/service/notification/NotificationRequest.java

```diff
@@ -5,15 +5,13 @@
 /**
  * Represents a request for sending an email notification.
  *
- * @param addressesTo   a list of primary recipients for the email. Each entry in the list contains
- *                      a {@link NameAndEmail} object with a name and email address.
- * @param subject       the subject of the email, represented by an {@link EmailSubject} object
- *                      that defines the email topic.
- * @param mailArguments a list of arguments for formatting the email body. Typically, these are
- *                      placeholders for dynamic content that will be injected into the email template.
+ * @param addressesTo   a list of recipients for the email. Each entry in the list contains
+ *                      a string with an email address.
+ * @param subject       the subject of the email, represented by a string that defines the email topic.
+ * @param text      a string for formatting the email body.
  */
 public record NotificationRequest (
-    List<NameAndEmail> addressesTo,
-    EmailSubject subject,
-    List<?> mailArguments
+    List<String> addressesTo,
+    String subject,
+    String text
```

Comment:
Replaced NotificationRequest class fields (object of the NameAndEmail, EmailSubject and unknown classes) with a String object